submissions/tests/test_api.py

```diff
@@ -283,13 +283,18 @@ def test_get_scores(self):
             )
 
     def test_get_top_submissions(self):
-        student_item = copy.deepcopy(STUDENT_ITEM)
-        student_item["course_id"] = "get_scores_course"
-        student_item["item_id"] = "i4x://a/b/c/s1"
+        student_item_1 = copy.deepcopy(STUDENT_ITEM)
+        student_item_1['student_id'] = 'Tim'
+
+        student_item_2 = copy.deepcopy(STUDENT_ITEM)
+        student_item_2['student_id'] = 'Bob'
 
-        student_1 = api.create_submission(student_item, "Hello World")
-        student_2 = api.create_submission(student_item, "Hello World")
-        student_3 = api.create_submission(student_item, "Hello World")
+        student_item_3 = copy.deepcopy(STUDENT_ITEM)
+        student_item_3['student_id'] = 'Li'
+
+        student_1 = api.create_submission(student_item_1, "Hello World")
+        student_2 = api.create_submission(student_item_2, "Hello World")
+        student_3 = api.create_submission(student_item_3, "Hello World")
```

Comment:
Can you also add multiple submissions for one student and ensure that only the latest one shows?